src/main/java/entities/Message.java

```diff
@@ -0,0 +1,59 @@
+package entities;
+
+import java.time.LocalDateTime;
+
+/**
+ * Message (abstract class) containing the message content and associated metadata
+ * The type of the message content is dependent on which child(ren) of Message the message is
+ * The metadata is universal for all message types and includes the ID of the sender, timestamp, ID of message
+ */
+public abstract class Message {
```

Comment:
I think it's a good idea that you made Message an abstract class just in case we would want to add other message types like voice or files